frontend/src/container/OnboardingContainer/Onboarding.styles.scss

```diff
@@ -80,6 +72,9 @@
 	overflow: hidden;
 	text-overflow: ellipsis;
 	text-align: center;
+
+	border-bottom: 1px solid #303030;
```

Comment:
here as well if present in design system

Reply:
didn't found any matches